pages/common/magick.md

```diff
@@ -0,0 +1,24 @@
+# ImageMagick
```

Comment:
Please make this all small letters.

Reply:
Too late now it seems, this is already merged. Should another PR be opened to fix it?

Reply:
Yes, that is the process we follow.